### PR TITLE
Fixed the relative path

### PR DESCRIPTION
**Pull request by SaiRev0**
Fixed the relative path as it was not correct

### CODE REVIEW

File: docs/volto/override-components.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ myst:
 
 When we want to override a specific file we can create an alias pointing to our own theme.
 
-So for example if we want to replace the logo, which is located in Volto at `components/theme/Logo/Logo.svg`,
+So for example if we want to replace the logo, which is located in Volto at `omelette/src/components/theme/Logo/Logo.svg`
```
Comment (stevepiercy) [inline docs/volto/override-components.md:18]:
```suggestion
So for example if we want to replace the logo, which is located in Volto at `omelette/src/components/theme/Logo/Logo.svg`,
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Restore deleted comma.

**Review by stevepiercy — APPROVED** [review]
Thank you! If no one from the Volto team on Discord answers my call to review this PR within a few days, then ping me to just merge it.

**Comment by ksuess** [comment]
Thank you @SaiRev0 for reporting and even fixing!

**Comment by SaiRev0** [comment]
@ksuess No Worrie always ready to help and to give back to the community that has helped me a lot.

**Pull request merged**